### PR TITLE
[DTRA] Bala/Fix scaling issue

### PR DESCRIPTION
**Pull request by balakrishna-deriv**
Changelog:
 
 The current charts expects the granularity of the ticks. So, we are passing the granularity based on the symbol.
 
 ---------------------------

### CODE REVIEW

File: chart_app/lib/src/chart_app.dart [hunk 1]
```diff
@@ -112,14 +112,22 @@ class ChartApp {
...
   int getQuotesInterval() {
-    if (feedModel.isFeedLoaded && feedModel.ticks.length > 1) {
-      final Tick previousTick = feedModel.ticks[feedModel.ticks.length - 2];
-      final Tick lastTick = feedModel.ticks.last;
-      if (previousTick.epoch != lastTick.epoch) {
-        return feedModel.ticks.last.epoch - previousTick.epoch;
-      }
+    final int granularity = configModel.granularity ?? 1000;
+
+    // The current charts expects the granularity of the interval.
```
Comment (balakrishna-deriv) [inline chart_app/lib/src/chart_app.dart:117]:
```suggestion
    // The current charts expect the granularity of the ticks.
```

File: chart_app/lib/src/chart_app.dart [hunk 2]
```diff
@@ -112,14 +112,22 @@ class ChartApp {
...
+    // The current charts expect the granularity of the ticks.
+    // Sometimes the feed misses a tick and the chart zoom
+    // doesn't work properly.
+    // The 2 tick symbols are hard coded here to fix the scaling issue.
+    // To do: Make flutter chart independent of the granularity.
+    // Flutter chart should do the x-axis calculations from the first epoch
+    // and last epoch.
+    final RegExp regex = RegExp(r'^(RDBEAR|RDBULL|R_)');
+
+    if (granularity == 1000 && regex.hasMatch(configModel.symbol)) {
```
Comment (kate-deriv) [inline chart_app/lib/src/chart_app.dart:126]: `configModel.symbol` is always in English?

**Review by kate-deriv — APPROVED** [review]

File: chart_app/lib/src/chart_app.dart (hunk 2 above)
Comment (balakrishna-deriv) [reply chart_app/lib/src/chart_app.dart:126]: yes :)

Comment (maryia-deriv) [reply chart_app/lib/src/chart_app.dart:126]: yes, `symbol` is always unchanged, e.g. `R_10`, `1HZ100V` :)

**Review by maryia-deriv — APPROVED** [review]

**Review by henry-deriv — APPROVED** [review]

**Pull request merged**